From ce76327a2bf278e94a0d14bc12e1725fad17b3c2 Mon Sep 17 00:00:00 2001
From: "Tobias C. Berner" <tcberner@FreeBSD.org>
Date: Sun, 24 Dec 2017 11:18:39 +0100
Subject: [PATCH] Add the values of the Inotify find call to the appropriate
 lists

* Add ${Inotify_INCLUDE_DIRS} to the viewer targets includes
* Add ${Inotify_LIBRARIES} to the viewer targets libraries
---
 messageviewer/src/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/messageviewer/src/CMakeLists.txt b/messageviewer/src/CMakeLists.txt
index 2301d75c..917439e5 100644
--- messageviewer/src/CMakeLists.txt
+++ messageviewer/src/CMakeLists.txt
@@ -203,7 +203,7 @@ add_library(KF5MessageViewer ${libmessageviewer_SRCS})
 generate_export_header(KF5MessageViewer BASE_NAME messageviewer)
 add_library(KF5::MessageViewer ALIAS KF5MessageViewer)
 
-target_include_directories(KF5MessageViewer INTERFACE "$<INSTALL_INTERFACE:${KDE_INSTALL_INCLUDEDIR_KF5}/MessageViewer/;${KDE_INSTALL_INCLUDEDIR_KF5}/messageviewer>")
+target_include_directories(KF5MessageViewer INTERFACE "$<INSTALL_INTERFACE:${KDE_INSTALL_INCLUDEDIR_KF5}/MessageViewer/;${KDE_INSTALL_INCLUDEDIR_KF5}/messageviewer>;${Inotify_INCLUDE_DIRS}")
 
 target_link_libraries(KF5MessageViewer
     PUBLIC
@@ -238,6 +238,7 @@ target_link_libraries(KF5MessageViewer
     Grantlee5::Templates
     Qt5::PrintSupport
     QGpgme
+    ${Inotify_LIBRARIES}
     )
 
 set_target_properties(KF5MessageViewer PROPERTIES
-- 
2.15.1

